solara/components/slider.py

```diff
@@ -120,6 +130,11 @@ def set_value_cast(value):
         v1, v2 = value
         reactive_value.set((int(v1), int(v2)))
 
+    if show_end_ticks:
+        ticks = [str(i) for i in np.arange(min, max, step) if i == min or i == max]
```

Comment:
I'm not sure this code is correct since this is  the same as [str(min), str(max)].
You mean sth like `[str(v) if i == min or v == max else '' for v in range(min, max+step/2, step) ]

Although I'm not sure with floats this will work (v might never equal max)

Reply:
You're right, it was a bad refactor 😓

Reply:
No problem, better get the idea out first. In the end I like it, now we can worry about code quality :)
Would be nice to see this particular feature tested btw.